Ovs 3755 sticky snapshot

eduardBM opened the pull request.

eduardBM committed:
OVS-3755 Implement Sticky snapshots

eduardBM committed:
OVS-3755 Implement Sticky snapshots
- use locales

eduardBM committed:
OVS-3755 Implement Sticky snapshots
- cleanup

khenderick commented on webapps/frontend/app/viewmodels/site/vdisk-detail.js:
```diff
@@ -162,6 +163,12 @@ define([
             self.loadStorageRouters();
         };
+        self.removeSnapshot = function(snapshotid) {
+            api.post('vdisks/' + self.vDisk().guid() + '/removesnapshot', {
```
I'd like to have a confirmation dialog here. There are a bunch of these around the code so you should have something to base on

khenderick commented on webapps/frontend/app/views/site/vdisk-detail.html:
```diff
@@ -159,6 +159,16 @@ <h4 data-i18n="ovs:vdisks.detail.snapshots"></h4>
                             <td><span class="fa pointer" data-bind="translate: (is_consistent ? 'ovs:icons.consistent' : 'ovs:icons.inconsistent'),
                                                                     tooltip: (is_consistent ? 'ovs:generic.consistentss' : 'ovs:generic.inconsistentss')"></span></td>
+                            <td><span data-bind="text: (is_sticky ? 'ovs:generic.sticky' : '-')"></span></td>
+                            <td>
```
I don't see a column for this td containing the remove button.

khenderick commented on ovs/lib/scheduledtask.py:
```diff
@@ -191,6 +196,8 @@ def make_timestamp(offset):
             for bucket in bucket_chain:
                 for snapshot in bucket['snapshots']:
+                    if snapshot.get('is_sticky') is True:
+                        continue
```
There shouldn't be any sticky snapshots in here, based on the filters above, right?

eduardBM committed:
OVS-3755 Implement Sticky snapshots
- gui cleanup (task confirmation, show sticky in vmachine snapshots)
- scheduledtask cleanup

eduardBM committed:
OVS-3755 Implement Sticky snapshots
- GUI improvements (confirmation on remove)
